capstone/src/main/webapp/styles/group.css

```diff
@@ -299,17 +308,19 @@ p {
   font-weight: normal;
   font-size: 24px;
   line-height: 28px;
-  padding: 8px 12px;
+  padding: 10px 15px;
+  margin-block-end: 0;
+  margin-block-start: 0;
 }
 
 #comment-input {
-  background-color: #eeeeee;
+  background-color: #EEEEEE;
   border-radius: 10px;
   align-content: left;
   margin-left: 100px;
   height: 40px;
   width: 70%;
-  color: #8a8686;
+  color: #8A8686;
```

Comment:
since we've set up variables for these colors I think it would make sense to use those? `var(--placeholder-gray)` and `var(--text-gray)`